src/_realizeCustomElements.ts

```diff
@@ -286,16 +294,25 @@ export default function realizeCustomElements(registry: CombinedRegistry, root:
 					else {
 						promise = Promise.all<any>([
```

Comment:
I think you can use a tuple type here to make the `.then()` destructuring work better.